engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/ExecutionStore.scala

```diff
@@ -3,11 +3,15 @@ package cromwell.engine.workflow.lifecycle.execution
 import cromwell.backend.BackendJobDescriptorKey
 import cromwell.core.ExecutionStatus._
 import cromwell.core.{CallKey, JobKey}
+import cromwell.engine.workflow.lifecycle.execution.ExecutionStore.FqnIndex
 import cromwell.engine.workflow.lifecycle.execution.WorkflowExecutionActor.{apply => _, _}
 import wdl4s._
 
+import scala.language.postfixOps
 
 object ExecutionStore {
+  type FqnIndex = (String, Option[Int])
```

Comment:
Yeah, that's why I gave you the parenthetical out :) Since you have the fancy benchmark tool I'd be curious to see what sort of impact it had. I suspect it'd be quite a lot actually.